rebasehelper/tests/test_specfile.py

```diff
@@ -79,20 +81,43 @@ def test_set_version(self, spec_object):
         spec_object.save()
         assert spec_object.header.version == NEW_VERSION
 
-    def test__write_spec_content(self, spec_object):
+    @pytest.mark.parametrize('spec_attributes, content', [
+        (
+            {
+                'path': SPEC_FILE,
+            },
+            'testing line1\ntesting line 2\n'
+        )
+    ])
+    def test__write_spec_content(self, mocked_spec_object, content):
```

Comment:
This makes me wonder - why do we test this? I mean, what does it actually test? `SpecContent.__str__()` or `file.write()`? I don't think the latter needs to be tested, and the former is already being tested by `TestSpecContent.test_string_representation()`.

Reply:
I think that the original intention was to check whether replacing the whole `SpecContent` has an effect on writing the spec. You are right that there is probably no point in testing a method which just writes a string representation of `SpecContent` (which is already covered by tests). I will remove the test.